pkg/controller/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -246,6 +279,22 @@ func (r *ReconcilePublishingStrategy) Reconcile(request reconcile.Request) (reco
 		}
 	}
 
+	// Delete all IngressControllers that are owned by cloud-ingress-operator but not in PublishingStrategy
+	for ingress, exists := range ownedIngressExistingMap {
+		if !exists {
```

Comment:
```suggestion
	for ingress, inPublishingStrategy := range ownedIngressExistingMap {
		if !inPublishingStrategy {
```
I think we could rename this variable to make the criteria for this `if` block more clear.

Reply:
+1